src/components/templates/PartyMap/PartyMap.tsx

```diff
@@ -31,6 +31,8 @@ export const PartyMap: React.FC = () => {
   const hasSelectedRoom = !!selectedRoom;
 
   const selectRoom = useCallback((room: Room) => {
+    if (room.type === RoomTypes.unclickable) return;
+
```

Comment:
This logic shouldn't be handled here, only the room component should care about it IMO. See my other comment in this PR about not using this + the other callbacks when room is unclickable.

Reply:
@0xdevalias we have keyboard controls, which are out of the Room component.

Reply:
That's fair, can keep this.

But the comments I made about the logic changes within the Room component should still be done as well.